.github/workflows/docker-publish.yml

```
@@ -0,0 +1,52 @@
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and Push Images
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          cache-from: type=registry,ref=ghcr.io/viscalyx/devshell-dsc:buildcache
++         cache-to: type=registry,ref=ghcr.io/viscalyx/devshell-dsc:buildcache,mode=max
+          platforms: linux/amd64,linux/arm64
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 46: mapping values are not allowed in this context

(syntax-check)